ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/ConfigEnricher.scala

```diff
@@ -32,17 +34,26 @@ class ConfigEnricher(allocatedParties: AllocatedParties) {
       case config: TransactionsStreamConfig =>
         config
           .copy(
-            filters = enrichFilters(config.filters)
+            filters = enrichFilters(config.filters) ++ convertFilterByPartySet(
```

Comment:
This appears to be done three times, may be nicer to extract this into a method that takes a config and returns an `enriched` config.

Reply:
That won't work as the config type is different in each case.
I could move the common fields to the parent trait shared by these three configs, but there's a fourth config (`CompletionsStreamConfig`) for which these filters don't make sense.

This could be worked around but I think overall it's simpler to have this repeated three times here